kmich

```diff
@@ -21,7 +21,7 @@ if [[ ! -f "${k_release_dir}/bin/kompile" ]]; then
     fi
 fi
 
-export PATH="$scriptpath/lib:$k_release_dir/bin:$PATH"
+PATH="$scriptpath/lib:$k_release_dir/bin:$PATH"
```

Comment:
????

Reply:
I want to export the PATH only once after getting the location of `parser_PGM`. The export moved farther down the file.